README.md

```diff
@@ -1,8 +1,8 @@
 Sparse OPTimisation Library
 ===========================
 
-<!-- [![Build Status](https://travis-ci.com/astro-informatics/sopt.svg?branch=development)](https://travis-ci.com/astro-informatics/sopt) -->
-[![build](https://img.shields.io/badge/build-passing-brightgreen.svg?style=flat)](https://travis-ci.com/astro-informatics/sopt)
+<!-- [![Build Status](https://travis-ci.com/astro-informatics/sopt.svg?branch=development)](https://github.com/astro-informatics/sopt/actions?query=workflow%3Acmake) -->
+![build](https://github.com/astro-informatics/sopt/actions/workflows/cmake.yml/badge.svg?branch=development)
```

Comment:
```suggestion
[![build](https://github.com/astro-informatics/sopt/actions/workflows/cmake.yml/badge.svg?branch=development)](https://github.com/astro-informatics/sopt/actions/workflows/cmake.yml?query=branch%3Adevelopment+)
```

Linked it to the Actions page for the `development` branch so show the status of all recent builds, rather than just showing the badge when you click on it.

Reply:
Great, thanks!